openpgp/packet/aead_config.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (C) 2019 ProtonTech AG
+
+package packet
+
+import "math/bits"
+
+// Only currently defined version
+const aeadEncryptedVersion = 1
+
+// AEADConfig collects a number of AEAD parameters along with sensible defaults.
+// A nil AEADConfig is valid and results in all default values.
+type AEADConfig struct {
+	// The AEAD mode of operation.
+	DefaultMode AEADMode
+	// Amount of octets in each chunk of data
+	ChunkSize uint64
+}
+
+var defaultAEADConfig = &AEADConfig{
+	DefaultMode: AEADModeEAX,
+	ChunkSize:   1 << 18, // 262144 bytes
+}
```

Comment:
(Also, there may be other benefits of having a single `DefaultConfig` exported instead of having to call each function in order to learn the defaults)

Reply:
Perhaps, but that's something we should think about in another PR then, rather than this one, no?

Reply:
+ 1